rcgen/src/key_pair.rs

```diff
@@ -101,12 +101,25 @@ impl KeyPair {
 					serialized_der: key_pair_serialized,
 				})
 			},
+			#[cfg(all(feature = "aws_lc_rs", not(feature = "ring")))]
+			SignAlgo::Rsa(sign_alg) => {
+				use aws_lc_rs::encoding::AsDer;
+				let key_pair_doc =
+					RsaKeyPair::generate(crate::ring_like::rsa::KeySize::Rsa2048)._err()?;
```

Comment:
Should there be a way to specify key size? I suspect there will be interest in 3072 bit keys at some point.

Reply:
Yeah the key size should be customizeable, but I'm not sure how to best expose that in the API. The `SignatureAlgorithm`s are independent of key size, and passing a parameter for all signature algorithms is also not great. Maybe a separate function that takes a `SignatureAlgorithm` plus a key size parameter and that panics if you pass it something non-RSA? IDK.

In general, the switch from 2048 to 4096 isn't *that* big of an improvement, it's not the square amount of computational complexity as one would think.